week-1/Homework/mandatory/3-function-output.js

```diff
@@ -1,16 +1,20 @@
 // Add comments to explain what this function does. You're meant to use Google!
+//Here Math.random() returns a random number between 0 and 10....eg 5.156564545
```

Comment:
Math.random returns a floating-point (decimal) random number, from 0 to 1 (inclusive of 0, excluding 1). In this function, it will return between 0 to 9.9999.